packages/fx-core/src/plugins/solution/fx-solution/v3/addFeature.ts

```diff
@@ -22,20 +22,31 @@ import { selectSingleFeatureQuestion } from "../../utils/questions";
 import arm from "../arm";
 import { BuiltInFeaturePluginNames } from "./constants";
 import { ensureSolutionSettings } from "../utils/solutionSettingsHelper";
+import { ProgrammingLanguageQuestion } from "../../../../core/question";
 
 function getAllFeaturePlugins(): v3.FeaturePlugin[] {
   return [
     Container.get<v3.FeaturePlugin>(BuiltInFeaturePluginNames.frontend),
     Container.get<v3.FeaturePlugin>(BuiltInFeaturePluginNames.aad),
+    Container.get<v3.FeaturePlugin>(BuiltInFeaturePluginNames.function),
+    Container.get<v3.FeaturePlugin>(BuiltInFeaturePluginNames.apim),
+    Container.get<v3.FeaturePlugin>(BuiltInFeaturePluginNames.keyVault),
+    Container.get<v3.FeaturePlugin>(BuiltInFeaturePluginNames.identity),
+    Container.get<v3.FeaturePlugin>(BuiltInFeaturePluginNames.sql),
   ];
 }
 
 export async function getQuestionsForAddFeature(
   ctx: v2.Context,
   inputs: v2.InputsWithProjectPath
 ): Promise<Result<QTreeNode | undefined, FxError>> {
+  const node = new QTreeNode({ type: "group" });
   const plugins = getAllFeaturePlugins();
   const featureNode = new QTreeNode(selectSingleFeatureQuestion);
+  if (!ctx.projectSetting.solutionSettings?.programmingLanguage) {
```

Comment:
Is this question added twice(both in core and solution)?

Reply:
programmingLanguage is now core-level question, solution will not ask it.